src/nrtk_explorer/library/dataset.py

```diff
@@ -24,3 +25,20 @@ class Dataset(TypedDict):
     categories: List[DatasetCategory]
     images: List[DatasetImage]
     annotations: List[DatasetAnnotation]
+
+
+def loadDataset(path: str) -> Dataset:
+    with open(path) as f:
+        return json.load(f)
+
+
+datasetJson: Dataset = {"categories": [], "images": [], "annotations": []}
+_datasetPath: str = ""
+
+
+def getDataset(path: str) -> Dataset:
```

Comment:
add a force arg here

Reply:
What do you mean?  `string_path = str(path)`?